identity/did.go

```diff
@@ -211,10 +211,10 @@ type Service interface {
 	GetKey(did DID, key [32]byte) (*KeyResponse, error)
 
 	// RawExecute calls the execute method on the identity contract
-	RawExecute(ctx context.Context, to common.Address, data []byte, gasLimit uint64) (txID IDTX, done chan bool, err error)
+	RawExecute(ctx context.Context, to common.Address, data []byte, gasLimit uint64) (txID IDTX, done chan error, err error)
```

Comment:
please review variable names for the changed type

Reply:
Can you suggest a name that fits the meaning of job completion and holds an error if triggered?